content/en/docs/getting-started/installation.md

```diff
@@ -16,14 +16,15 @@ This document describes how to install OpenFunction.
 
 | OpenFunction Version | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20+ |
 | -------------------- | --------------- | --------------- | --------------- | ---------------- |
+| v0.6.0               | √               | √               | √               | √                |
 | v0.5.0               | √               | √               | √               | √                |
 | v0.4.0               | √               | √               | √               | √                |
 | v0.3.x               | √               | √               | √               | √                |
```

Comment:
v0.3.x could be removed

Reply:
[Removed](https://github.com/OpenFunction/website/pull/45/commits/8075b7e480bbab95ab4a9e6225aeff503044188b).